packages/core/src/utils.js

```diff
@@ -1265,3 +1265,13 @@ export function schemaRequiresTrueValue(schema) {
 
   return false;
 }
+
+export const descriptionId = id => `${id}__description`;
+
+export const helpId = id => `${id}__help`;
+
+export const errorsId = id => `${id}__errors`;
+
+export const ariaDescribedBy = id => ({
+  "aria-describedby": `${errorsId(id)} ${descriptionId(id)} ${helpId(id)}`,
```

Comment:
For consistency, I wonder if it's better to just return a string here instead of a dict.

Reply:
Done! You are right. It's better that way.